code/style.css

```diff
@@ -22,37 +35,63 @@ p {
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background-image: linear-gradient(160deg, #1a1b1c 0%, #0e4c9a 100%);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+  -webkit-box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  -moz-box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  transition-duration: 0.25s; 
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background-image: linear-gradient(160deg, #1a1b1c 0%, #0b4e25 100%);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+  -webkit-box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  -moz-box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  transition-duration: 0.25s; 
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background-image: linear-gradient(160deg, #1a1b1c 0%, #ad1641 100%);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+  -webkit-box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  -moz-box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
+  box-shadow: 0px 10px 19px 0px rgba(189,189,189,1);
```

Comment:
In general I like light code and styling that does the job. I think that you have achieved that. You also gave good tips on slack on how to make the cards gradient. I also like transition-duration! GJ!

Reply:
Agreed on the duration!